tlatools/org.lamport.tlatools/src/tlc2/TLC.java

```diff
@@ -1571,15 +1571,17 @@ private void printUsage()
         sharedArguments.add(new UsageGenerator.Argument("-dump", "format file",
                                                         "dump all states into the specified file.\n"
     													+ "The format parameter can be omitted, or it can be a comma-separated\n"
-    														+ "list beginning with 'dot' that may also contain one or both of\n"
-    														+ "the items 'colorize' and 'actionlabels'. If format is omitted,\n"
-    														+ "TLC writes a list of all reachable states, described by TLA+ formulas, \n"
-                                                            + "on file. Otherwise, TLC writes the state graph in dot format, \n"
+    														+ "list beginning with 'dot' that may also contain a subset of\n"
+    														+ "the items 'colorize', 'actionlabels', and `constrained. If format is\n"
```

Comment:
```suggestion
    														+ "the items 'colorize', 'actionlabels', and 'constrained'. If format is\n"
```
missing end backtick, using single quote for consistency :)

Reply:
thanks, should be good now.